utils/amt-integration-tests/tests/data_helpers.py

```diff
@@ -301,16 +296,41 @@ def load_grading_period(engine: engine.base.Engine, grading_period_table: str) -
         engine, grading_period_descriptor, DESCRIPTOR_NAMESPACE
     )
 
+    # Now add the grading period
+    if GRADING_PERIOD_KEY not in already_loaded.keys():
+        already_loaded[GRADING_PERIOD_KEY] = []
```

Comment:
Again, this is all about allowing a function to run multiple times - because the BACKGROUND data inserts run once for every scenario - without actually doing any work after the first execution.